R/deploymentTarget.R

```diff
@@ -68,3 +70,29 @@ createDeploymentTarget <- function(appName,
     server = server
   )
 }
+
+defaultAppName <- function(recordPath, server = NULL) {
+  if (isDocumentPath(recordPath)) {
+    name <- file_path_sans_ext(basename(recordPath))
+    if (name == "index") {
+      # deploying a website; use name of parent directory
```

Comment:
Why does `index` indicate a directory? My convention is to _ALWAYS_ use `index.Rmd`, with the directory as the meaningful name - even for single-document deploys.

The code feels right, but the comment might be misleading.